gdal/ogr/ogr_srs_esri_names.h

```diff
@@ -1,3 +1,78 @@
+static const char * const apszDatumNameMapping[] = {
```

Comment:
I'd suggest we put all these extra tables (there should be a lot more btw), in its own header file, so it is easy to understand which parts are esri custom, and which are gdal